### ISSUE TITLE
Good way to see build output?

### ISSUE THREAD

**Issue by ywpkwon** [fannheyward/coc-texlab#136]
When running `CocCommand latex.build`, how can we see the build output? I tried to see it through `CocInfo` but it has other reports, too (like coc-prettier, coc-snippets, etc). Moreover, since multiple `latex.build` outputs in the past building are also there without any separation, it was hard to see.  For example:
```
(...skip...)
!  ==> Fatal error occurred, no output PDF file produced!
Transcript written on experience.log.
Collected error summary (may duplicate other messages):
  pdflatex: Command for 'pdflatex' gave return code 256
Latexmk: Errors, so I did not complete making targets
Latexmk: Use the -f option to force complete processing,
 unless error was exceeding maximum runs of latex/pdflatex.
Latexmk: This is Latexmk, John Collins, 1 January 2015, version: 4.41.    (<-- this is new run)
Rule 'pdflatex': Rules & subrules not known to be previously run:
   pdflatex
Rule 'pdflatex': The following rules & subrules became out-of-date:
(...skip...)
```
I marked `<-- this is new run` above.
Is there way to see latest build output separately? This may be not a question for `coc-latex`. Is there a general output window `coc` shows with (and can I see latex build output from there)?

**Comment by mnabila** [comment]
you can try `:CocCommand workspace.showOutput` and select textlab from the list for showing coc-texlab build output
![Screenshot_20200318_065008](https://user-images.githubusercontent.com/37516576/76911929-c2e76c00-68e4-11ea-8ae7-9cb74df398be.png)

_--sorry if my english to bad_

**Comment by ywpkwon** [comment]
Thanks for your response! Yes, I tried that, too. It was good since it doesn't have any other coc messages. However, when I run builds multiple times, there was no boundaries (or timestamp) between build messages so it was hard to see where is the "latest" build message starts. (e.g., I had to search for `Latexmk: This`) Can I see only the output messages from the "latest" build?

-- No worries to your English
